packages/common/infra/src/livedata/index.ts

```diff
@@ -132,10 +132,100 @@ export class LiveData<T = unknown> implements InteropObservable<T> {
     return data;
   }
 
+  private static GLOBAL_COMPUTED_RECURSIVE_COUNT = 0;
+
+  /**
+   * @example
+   * ```ts
+   * const a = new LiveData('v1');
+   * const v1 = new LiveData(100);
+   * const v2 = new LiveData(200);
+   *
+   * const v = LiveData.computed(get => {
+   *   return get(a) === 'v1' ? get(v1) : get(v2);
+   * });
+   *
+   * expect(v.value).toBe(100);
+   * ```
+   */
+  static computed<T>(
```

Comment:
Will this compare computed res with previous? Since in some case, although some deps changed, but the computed res is same.

Reply:
currently no compare,

I think we can do compare here